cookiecutter/{{cookiecutter.github_repo_name}}/.github/RELEASE_NOTES.template.md

```diff
@@ -6,7 +6,7 @@
 
 ## Upgrading
 
-<!-- Here goes notes on how to upgrade from previous versions, including deprecations and what they should be replaced with --> 
+<!-- Here goes notes on how to upgrade from previous versions, including deprecations and what they should be replaced with -->
```

Comment:
"Here goes notes" reads a bit weird to me..

Reply:
"Here go notes"? :D

Reply:
Well, automerge was enabled, so maybe open an issue for it?

Reply:
"here go the notes on.."?

Reply:
I also feel "here" and "go" don't go well together.  It has to be "Here come the notes on..." or "There go the notes on..."

Reply:
How about just "Notes on how to upgrade.."?

Reply:
That was the first good one.  How about "Add notes here, on ..."?

Reply:
Also good, yes